scripts/env.sh

```diff
@@ -24,9 +24,11 @@ export NIMBUS_ENV_DIR="${ABS_PATH}"
 export GOPATH="${ABS_PATH}/../../go"
 export GO111MODULE=on
 
+export NIM_PATH=$(cd "${ABS_PATH}/../vendor/Nim/bin"; pwd)
```

Comment:
I guess this could have been more consistent if `NIM_PATH` was referring to the top-level Nim dir. We also use `DIR` and `PATH` across this file, perhaps we should pick one term and stick to it.